src/services/BooksService.ts

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { create } from 'apisauce';
+
+import { HEADERS } from '../constants/constants';
+
+interface BookPage {
```

Comment:
can you use this interfaces in this file 
**src/components/BookList/index.tsx**